doc/rtd/topics/debugging.rst

```diff
@@ -155,10 +164,13 @@ commandline:
 
 .. code-block:: shell-session
 
-  $ sudo cloud-init single --name cc_ssh --frequency always
-  ...
-  Generating public/private ed25519 key pair
-  ...
+   $ sudo cloud-init single --name cc_ssh --frequency always
+
+.. code-block::
+
+   ...
+   Generating public/private ed25519 key pair
+   ...
```

Comment:
The codeblocks starting on line 225 through the end of the document (in your copy of this file) has commands that are missing the `$` (the line containing `ssh ubuntu@$CI_VM_IP -- cloud-init status --wait --long`)

Reply:
That line doesn't have a $ in it on the live version of the docs either - if it should have one, I can add it in np